regression/cbmc/r_w_ok_inconsistent_integer/main.c

```diff
@@ -0,0 +1,22 @@
+#include <assert.h>
+#include <stdlib.h>
+
+char *p;
+
+void test_global_pointer()
+{
+  p = (size_t)123 << (sizeof(char *) * 8 - 1) | 123;
```

Comment:
:question: An explanation in line as to what this is trying to achieve would be useful. Does it need to be this complicated? Can you just go `p = 1234;`?

Reply:
Added a description. It's creating a pointer with object bits being 123 and the offset being 123 (also noticed that it should be -8 instead of -1).